extensions/wikia/AssetsManager/config.php

```diff
@@ -197,12 +197,6 @@
 		'//extensions/wikia/Lightbox/scripts/LightboxLoader.js',
 		'#group_imglzy_js',
 		'//extensions/wikia/AjaxLogin/AjaxLoginBindings.js',
-		// We allow logged in users to change login without logging out
-		'//extensions/wikia/UserLogin/js/UserLoginAjaxForm.js',
-		// We allow logged in users to create account without logging out
-		'//extensions/wikia/UserLogin/js/UserSignupAjaxForm.js',
```

Comment:
Moved this to the `user_signup_js` group (and renamed to `UserSignupAjaxValidation`)